### PR TITLE
Adding 3rd example to repo

### PR DESCRIPTION
**Pull request by vtimd**
This pull request will add the 3rd set of example code to the repo for the blog on how to use terraform outputs across stacks.

### CODE REVIEW

File: 03-terraform-outputs-across-stacks/stacks/EC2/config.tm.hcl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+globals {
+  "dependencies" = [
+    "stacks/vpc",
+  ]
+}
```
Comment (soerenmartius) [inline 03-terraform-outputs-across-stacks/stacks/EC2/config.tm.hcl:5]: We should always add a "newline at the end of each file", hence the stop sign here.

Here's how to do that with vscode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

And that's why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Please see the comments. Also too, did you test this code, does it actually run?

**Comment by soerenmartius** [comment]
@vtimd just tried the code locally, it unfortunately doesn't work (try run `terramate generate` to trigger the code generation)
Screenshot 2023-12-04 at 22 40 10

Keep in mind that the example we defined in last weeks session was never tested and was meant as an example for you how the final implementation could look like

**Comment by vtimd** [comment]
This PR was just a placeholder for the file links. I didn’t test the code. I’m going to get it fixed in parallel while I write the blog. I just needed the GitHub links for the blog. I knew the code was wrong when I saved it in Code. It was all red 😂

File: 03-terraform-outputs-across-stacks/stacks/EC2/config.tm.hcl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+globals {
+  "dependencies" = [
```
Comment (kkmlr) [inline 03-terraform-outputs-across-stacks/stacks/EC2/config.tm.hcl:2]:
```suggestion
  dependencies = [
```

File: 03-terraform-outputs-across-stacks/stacks/EC2/dependencies.tm.hcl [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+          bucket  = "mybucket"
+          region  = "us-east-1"
+          encrypt = true
+          key     = "${value.value}/terraform.state"
+        }
+      }
+    }
+   }
+  }
+}
```
Comment (kkmlr) [inline 03-terraform-outputs-across-stacks/stacks/EC2/dependencies.tm.hcl:19]:
```suggestion
      }
    }
  }
}
```
fixes `An argument or block definition is required here.`

File: 03-terraform-outputs-across-stacks/stacks/EC2/dependencies.tm.hcl [hunk 4]
```diff
@@ -0,0 +1,18 @@
+generate_hcl "dependencies.tf" {
+  content {
+    tm_dynamic "terraform_remote_state" "vpc" {
+      for_each = global.dependencies
+
```
Comment (kkmlr) [inline 03-terraform-outputs-across-stacks/stacks/EC2/dependencies.tm.hcl:5]:
```suggestion
    tm_dynamic "terraform_remote_state" {
      for_each = global.dependencies
      iterator = value
      labels   = ["vpc"]


```

**Review by soerenmartius — CHANGES_REQUESTED** [review]
left more comments in shortcut

File: 03-terraform-outputs-across-stacks/stacks/config.tm.hcl [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+generate_hcl "backend.tf" {
+  content {
+    terraform {
+      backend "s3" {
+      bucket = "mybucket"
+      key    = "${terramate.stack.path.relative}/terraform.state"
+      region = "us-east-1"
+      encrypt = true
+     }
+  }
```
Comment (soerenmartius) [inline 03-terraform-outputs-across-stacks/stacks/config.tm.hcl:11]: formatting issue (run `terramate fmt`)

**Comment by vtimd** [comment]
Closing. Starting Fresh.

**Pull request closed without merging**